script.js

```diff
@@ -0,0 +1,90 @@
+// define books as a collection of arrays
+let books = [
+  {
+    id: 1,
+    title: 'Lorem',
+    author: 'Testeroo Testyy',
+  },
+  {
+    id: 2,
+    title: 'Ipsum',
+    author: 'Testeroo Testyy',
+  },
+  {
+    id: 3,
+    title: 'Dolor',
+    author: 'Testeroo Testyy',
+  },
+];
+
+// Get DOM elements
+const bookListElement = document.getElementById('book-list');
+const addButtonElement = document.getElementById('add-btn');
+const titleInputElement = document.getElementById('title-input');
+const authorInputElement = document.getElementById('author-input');
+
+function addBook(title, author) {
+  const book = {
+    id: books.length + 1,
+    title,
+    author,
+  };
+  books.push(book);
+  localStorage.setItem('books', JSON.stringify(books));
```

Comment:
- [OPTIONAL] Great job so far 👏🏻 However, I will recommend the use of array methods that don't mutate the array, e.g. [concat](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/concat), [filter](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/filter), [slice](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/slice) instead of [push](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/push), [pop](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/pop), [splice](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/splice) and so on.
 You may visit this link: [Immutable Array Methods](https://www.sitepoint.com/immutable-array-methods-javascript/) for more documentation